kwanj-k/storemanager-API
# #161404660 Add version two API tests

### What does this PR do?
Adds tests for the version two of the API 

#### Description of Task to be Completed ?
Write tests for: 
-- Add signup endpoint and login
-- Add/get and delete product endpoint
-- Add admin and attendant endpoint

### How should this be manually tested?
* After cloning the repo,git pull the branch ft-v2tests-#161404660
* Install a virtual enviroment and activate it [Refer here](https://docs.python-guide.org/dev/virtualenvs/)
* Pip install the the requirements.txt
* Run the following:

  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```
   ``` pytest --cov-report term-missing --cov=app ```

The new 14 tests should fail as the functionality is not there

![screenshot from 2018-10-23 13-00-55](https://user-images.githubusercontent.com/33450849/47352986-c791fb80-d6c3-11e8-883c-1af8021ee293.png)

### What are the relevant Pivotal Tracker stories?
 #161404660
